test/unit/helper.spec.js

```diff
@@ -817,4 +817,29 @@ describe('Helpers Unit Tests', () => {
       );
     });
   });
+
+  describe('getTimeTilAvailable()', () => {
+    it('should format blocks to human-readable time', () => {
+      const numBlocks = 463;
+      const time = helpers.getTimeTilAvailable(numBlocks);
+      expect(time, 'to equal', '3 days and 5 hours');
+    });
+
+    it('should show NaN for undefined', () => {
+      const time = helpers.getTimeTilAvailable(undefined);
+      expect(time, 'to equal', 'NaN days and NaN hours');
```

Comment:
We should probably throw an Error for `undefined`. This should only happen if the api changed and `blocksTilMaturity` isn't set as expected. And we wanna get notified in that case.

Reply:
fixed